sql/core/src/test/scala/org/apache/spark/sql/execution/ui/UISeleniumSuite.scala

```diff
@@ -102,13 +104,13 @@ class UISeleniumSuite extends SparkFunSuite with WebBrowser {
     eventually(timeout(10.seconds), interval(100.milliseconds)) {
       val sd = findErrorMessageOnSQLUI()
       assert(sd.size === 1, "Analyze fail shall show the query in failed table")
-      assert(sd.head.startsWith("[TABLE_OR_VIEW_NOT_FOUND]"))
+      assert(sd.head.startsWith("TABLE_OR_VIEW_NOT_FOUND"))
 
       val id = findExecutionIDOnSQLUI()
       // check query detail page
       go to s"${spark.sparkContext.uiWebUrl.get}/SQL/execution/?id=$id"
       val planDot = findAll(cssSelector(""".dot-file""")).map(_.text).toList
-      assert(planDot.head.startsWith("digraph G {"))
+      assert(planDot.size === 1)
```

Comment:
Cannot we check the text like before?

Reply:
Yes, we cannot check the test because it's invisible text, @viirya . I verified it manually the following.

- The text is changed and now starting with ` digraph G {` instead of `digraph G`. 
- Since this is an error case check, the graph itself is empty. There is no meaning to check.